education.html

```diff
@@ -6,28 +6,44 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>Education</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />  
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />  
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" /> 
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/css/materialize.min.css"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"></script> 
+  <link rel="stylesheet" type="text/css" href="./css/style.css" />
   <link rel="stylesheet" type="text/css" href="./css/education.css" />
+  <script src="scripts/main.js"></script>
 </head>
 
 <body>
-  <header class="header">
-    <a href="" class="logo">logo</a>
-    <input class="menu-btn" type="checkbox" id="menu-btn" />
-    <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
-    <ul class="menu">
+  <nav>
+    <div class="nav-wrapper">
+      <a href="#" class="brand-logo">Logo</a>
+      <a href="#" class="sidenav-trigger" data-target="mobile-nav">
+        <i class="material-icons">menu</i>
+      </a>
+
+      <ul class="right hide-on-med-and-down">
         <li><a href="./index.html">Home</a></li>
         <li><a href="#">About</a></li>
         <li><a href="./experience.html">Experience</a></li>
         <li><a href="./projects.html">Projects</a></li>
-        <li><a href="./education.html">Education</a></li>
         <li><a href="#">Contact</a></li>
+        <li><a href="./education.html">Education</a></li>
```

Comment:
Why was this change made, you've done it on all the pages. In fact I feel that the Contact tab should be at the right most side

Reply:
okay, then I have done this according to the design of home page in xd file. Should I change it?